src/server/req_delete.c

```diff
@@ -119,6 +120,27 @@ static char *sigtj =  SIG_TermJob;
 static char *acct_fmt = "requestor=%s@%s";
 static int qdel_mail = 1; /* true: sending mail */
 
+/**
+ * @brief
+ *     Wrapper function that calls process_hooks()
+ *
+ * @return	int
+ */
+
+int
+caller_to_process_hooks(struct batch_request *preq, char *hook_msg, size_t msg_len,
```

Comment:
This function is very much similar to "call_to_process_hooks" in req_runjob.c. If our need is to log return value of process_hook then we should probably make the function more generic than creating another copy of it.